### DISCUSSION TITLE
Can add directorys with log files to output the files from several directorys?

### DISCUSSION THREAD

**Discussion by DockerComposeFiles** [opcodesio/log-viewer#134] [Ideas]
I it possible to set more folders to this menu, which located under /myapplicationlogs in the root system

![image](https://user-images.githubusercontent.com/87118951/195603537-41b6b396-49c2-4739-bd3d-84ca1f022aa4.png)

**Comment by arukompas** [comment]
@DockerComposeFiles yes, just add any paths (absolute or relative) in the `'include_files'` part of the configuration:

```php
// config/log-viewer.php

    'include_files' => [
        '*.log',
        '/myapplicationlogs/*.log',
    ],
```
